src/core/sock/sockinfo_tcp.cpp

```diff
@@ -1154,34 +1138,12 @@ bool sockinfo_tcp::prepare_dst_to_send(bool is_accepted_socket /* = false */)
             m_pcb.tso.max_send_sge = ring->get_max_send_sge();
         }
     }
-    if (ret_val && is_xlio_socket()) {
+    if (ret_val && m_p_group) {
         add_tx_ring_to_group();
-    } else if (ret_val && m_entity_context) {
-        // Add the rings of the incoming socket to the entity context.
-        add_tx_ring_to_entity_context();
     }
     return ret_val;
 }
 
-bool sockinfo_tcp::prepare_dst_to_send_entity_context()
```

Comment:
We should remove then dst:_entry::prepare_to_send_entity_context as well

Reply:
Removed and fixed ring resolution with subsequent run of prepare_to_send()